docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM nikolaik/python-nodejs:python3.9-nodejs15
+
+RUN wget -qO- https://github.com/strukturag/pdfdraw/archive/v0.1.2.tar.gz | tar -C /tmp -xvz --strip 1
```

Comment:
Could you please change this to use the files from the repository so the Docker container is not pinned to a certain version but runs the code checked out?

Maybe move the files from this PR into the `server` folder so you can use a workflow like this
```
git clone https://github.com/strukturag/pdfdraw.git
cd pdfdraw/server
docker-compose up -d
```

Reply:
good point, done it exactly how you recommended